build.gradle

```diff
@@ -7,12 +7,50 @@ description = 'RxJava: Reactive Extensions for the JVM – a library for composi
 
 apply plugin: 'rxjava-project'
 apply plugin: 'java'
+apply plugin: 'findbugs'
+apply plugin: 'jacoco'
 
 dependencies {
     testCompile 'junit:junit-dep:4.10'
     testCompile 'org.mockito:mockito-core:1.8.5'
 }
 
+////////////////////////////////////////////////////////////////////
+// to run findbugs: 
+//     ./gradlew findbugsMain
+// then open build/reports/findbugs/main.html
+////////////////////////////////////////////////////////////////////
+
+findbugs {
+  ignoreFailures = true
```

Comment:
Please noo, my initial intent — run code quality tools in all regular builds and fail them if one of the tool found some problem, some cases can be suppressed of course, but let's use everything to improve and verify code quality?

In our projects we even trying to enforce compiler warnings as errors (if it's possible, but some kind of warnings can not be suppressed…).